### PR TITLE
Replaced occurrences of 'ODK 1' to 'ODK'

### LINKED ISSUE #1021: Remove all instances of "ODK 1" from ODK-X docs
Initially requested at https://github.com/opendatakit/docs/pull/1018

### PR DESCRIPTION
**Pull request by ray-fung**
closes #1021

addresses #1018 

#### What is included in this PR?
As noted in #1018, this new PR replaces occurrences of ODK 1 to just ODK for clarification.

### CODE REVIEW

File: odk2-src/aggregate-tables-extension.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@ First you’ll have to install ODK Aggregate v1.4.15 to a server (see :doc:`aggr
   #. Add ODK Aggregate usernames by typing one or more users' e-mail addresses into the text area and clicking :guilabel:`Add User`.
   #. If you have created an ODK Aggregate username, be sure to :guilabel:`Change Password` on that account to set the initial password for the account.
   #. Grant these users the :guilabel:`Synchronize Tables` permissions.
-  #. Select at least one user to be the administrator and grant them :guilabel:`Administer Tables` permissions. This user will have the ability to :guilabel:`Reset App Server` from the Android device and add or remove tables and configuration files on the server. This is the equivalent of the Form Manager permissions in ODK 1.x deployments.
+  #. Select at least one user to be the administrator and grant them :guilabel:`Administer Tables` permissions. This user will have the ability to :guilabel:`Reset App Server` from the Android device and add or remove tables and configuration files on the server. This is the equivalent of the Form Manager permissions in ODK.x deployments.
```
Comment (yanokwa) [inline odk2-src/aggregate-tables-extension.rst]: This last sentence doesn't make sense.  Maybe replace "ODK.x" with "ODK".

Comment (ray-fung) [reply odk2-src/aggregate-tables-extension.rst]: This should be fixed now. Along with a couple of additional changes of ODK 2 to ODK-X (specifically the title).

File: odk2-src/index.rst [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
   geotagging
 
-Welcome to Open Data Kit 2's documentation!
+Welcome to Open Data Kit-X's documentation!
```
Comment (yanokwa) [inline odk2-src/index.rst]: I'd make this ODK-X.

Comment (ray-fung) [reply odk2-src/index.rst]: Fixed

**Pull request merged**